timemachine/potentials/bonded.py

```diff
@@ -68,7 +68,7 @@ def restraint(conf, lamb, params, lamb_flags, box, bond_idxs):
     return energy
 
 # lamb is *not used* it is used in the alchemical stuff after
-def harmonic_bond(conf, params, box, lamb, bond_idxs):
+def harmonic_bond(conf, params, box, lamb, bond_idxs, lamb_mult=None, lamb_offset=None):
     """
```

Comment:
Update Parameters in docstring to include

```
    lamb_mult: None, or broadcastable to bond_idxs.shape[0]
         prefactor = (lamb_offset + lamb_mult * lamb)

    lamb_offset: None, or broadcastable to bond_idxs.shape[0]
         prefactor = (lamb_offset + lamb_mult * lamb)
```

or similar

Reply:
will fix - i like the "broadcastable to bond_idxs.shape[0]" part